{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_alluredemo1.py:20: Exception"}, "start": 1657706278776, "stop": 1657706278777, "uuid": "d2fca980-b8d5-4ea4-9f21-7ad290a29d9b", "historyId": "62e3d0efc8226834b589024da8762ee8", "testCaseId": "17acedbab6e0d57f20e780d7d6084ef4", "fullName": "test_alluredemo1#test_broken", "labels": [{"name": "suite", "value": "test_alluredemo1"}, {"name": "host", "value": "tonghuan"}, {"name": "thread", "value": "17720-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alluredemo1"}]}